core/block/chats/service.go

```diff
@@ -260,14 +268,38 @@ func (s *service) AddMessage(ctx context.Context, sessionCtx session.Context, ch
 		return err
 	})
 	if err == nil {
-		go s.sendPushNotification(spaceId, chatObjectId, messageId, message.Message.Text)
+		spaceName := s.store.GetSpaceName(spaceId)
+		_, _, details := s.identityService.GetMyProfileDetails(ctx)
+		var senderName string
+		if details != nil {
+			senderName = details.GetString(bundle.RelationKeyName)
+		} else {
+			log.Error("chats/AddMessage: failed to get profile name, details are empty")
```

Comment:
`log` is already scoped to `chats` you can simply use string `"send push notification: ..."`